application/forms/Authentication/LoginForm.php

```diff
@@ -98,6 +111,22 @@ public function onSuccess()
         $authenticated = $authChain->authenticate($user, $password);
         if ($authenticated) {
             $auth->setAuthenticated($user);
+            if ($this->getElement('rememberme')->isChecked()) {
+                $rsa = new RSA();
+                $rsa->loadKey(...RSA::keygen());
+                $data = $rsa->encryptToBase64($user->getUsername(), $password);
+                $data[] = base64_encode($rsa->getPublicKey());
+
+                $values['username']     = $user->getUsername();
```

Comment:
Please don't align/indent the equal signs here. A single space after the parameter name is enough.